css/stanford_news_grid.css

```diff
@@ -0,0 +1,17 @@
+.views-grid-two .views-row {
```

Comment:
This seems a little too generic and could effect other views on the page. Can you make these selectors more specific to the view included in this pr? Thanks.